core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.component.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the questions tab.
+ */
+
+require(
```

Comment:
Are we only expecting to change the filename here? (If yes, can we use `git mv` so that it doesn't add this as a new file?)

Reply:
No, This was converted from an AngularJS directive to an AngularJS component.

Reply:
I don't see this file having any angular code? I only see the filenames are changed* am I missing anything?

Reply:
This https://github.com/oppia/oppia/blob/ab6b1fb9b33c507dd51817acf267c1b3e0de6d78/core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.directive.ts#L32-L44
Changes to this
https://github.com/oppia/oppia/blob/f10fb5a9c4d2fd44cc8408e02ffa853e7a62f272/core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.component.ts#L32-L37